drivers/sensor/fxos8700/Kconfig

```diff
@@ -12,6 +12,8 @@ menuconfig FXOS8700
 	help
 	  Enable driver for the FXOS8700 accelerometer/magnetometer
 
+if !HAS_DTS_I2C_DEVICE
+
```

Comment:
I can't comment on the proper line, but can you remove the !HAS_DTS_I2C, now that this is wrapped in a !HAS_DTS_I2C_DEVICE